app/views/cancelRegistration/CancelRegistrationDateView.scala.html

```diff
@@ -26,6 +28,13 @@
 
 @(form: Form[_], mode: Mode)(implicit request: Request[_], messages: Messages)
 
+@dateHintText = {
+    @messages("cancelRegistration.cancelRegistrationDate.hintWithin14Days")
```

Comment:
I feel like this should be in its own "< p >< /p >"

Reply:
Done